src/evict/evict_lru.c

```diff
@@ -2499,18 +2523,37 @@ __wt_page_evict_urgent(WT_SESSION_IMPL *session, WT_REF *ref)
     WT_ASSERT(session, !__wt_ref_is_root(ref));
 
     page = ref->page;
-    if (F_ISSET_ATOMIC_16(page, WT_PAGE_EVICT_LRU) || S2BT(session)->evict_disabled > 0)
+
+    if (S2BT(session)->evict_disabled > 0 || F_ISSET_ATOMIC_16(page, WT_PAGE_IN_URGENT_QUEUE))
         return (false);
 
-    /* Append to the urgent queue if we can. */
     cache = S2C(session)->cache;
+
```

Comment:
Extra line.

Reply:
Updated in [db8e365](https://github.com/wiredtiger/wiredtiger/pull/8950/commits/db8e3651cf54d1a0cff7ba858f6ee4241da64dcf)